spinnaker/resource_pipeline.go

```diff
@@ -14,16 +14,22 @@ func resourcePipeline() *schema.Resource {
 		Schema: map[string]*schema.Schema{
 			"application": {
 				Type:     schema.TypeString,
+				ForceNew: true,
 				Required: true,
 			},
 			"name": {
 				Type:     schema.TypeString,
+				ForceNew: true,
 				Required: true,
 			},
 			"pipeline": {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"pipeline_id": {
```

Comment:
I'm not familiar with with this property. Does this link a pipeline to a parent template?

Reply:
It doesn't, this is a unique UUID that identifies the pipeline in spinnaker's backend. It's only needed for the UpdatePipeline call I believe